databricks/koalas/tests/test_series.py

```diff
@@ -1824,6 +1824,35 @@ def test_squeeze(self):
         kser = ks.from_pandas(pser)
         self.assert_eq(kser.squeeze(), pser.squeeze())
 
+    def test_swaplevel(self):
+        # MultiIndex with two levels
```

Comment:
What if the index is not multi index?

Reply:
`IndexError: Too many levels: Index of the series has only 1 levels, -2 is not a valid level number`.

That's a good test case! Let me add it.